charts/kyverno/values.yaml

```diff
@@ -48,7 +48,14 @@ podAnnotations: {}
 
 podSecurityContext: {}
 
-affinity: {}
+# By default this will make sure two pods don't end up on the same node
+# Changing this to a region would allow you to spread pods across regions
+antiAffinityTopologyKey: "kubernetes.io/hostname"
+
+# Hard means that by default pods will only be scheduled if there are enough nodes for them
+# and that they will never end up on the same node. Setting this to soft will do this "best effort"
+antiAffinity: "hard"
```

Comment:
I don't see where this is getting used.

Reply:
Removed it. The current code will set it to `hard` by default, right?